ci-operator/config/openshift/dedicated-admin-operator/openshift-dedicated-admin-operator-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+binary_build_commands: make build
+canonical_go_repository: github.com/openshift/dedicated-admin-operator
+images:
+- from: base
+  to: dedicated-admin-operator
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+  tag: 'osd'
+  tag_overrides: {}
```

Comment:
It's the first time I see `osd` in this stanza, what is the intention? We [recently removed](https://github.com/openshift/ci-operator/pull/286) the `tag` support, it was never used...

Pretty much openshift images have this `tag_specification`, how are you different? 

```
tag_specification:                                                                                                                                                                        
  name: '4.0'
  namespace: ocp
```

Reply:
Since the dedicated-admin-operator will be used only for OSD (Openshift Dedicated) I tagged it differently.
With this tag I wanted to make sure this operator would not be included into the main ocp 4 release image.

Reply:
You'll want to use a separate namespace/imagestream instead. Perhaps

```yaml
namespace: openshift
name: osd
```

Reply:
@smarterclayton might be able to advise the proper promotion place for this case. Without a `promotion` stanza, the built images won't be promoted anywhere, it's safe to have a ci-operator config with `tag_specification` & `base_images` pointing to the `ocp` namespace.